pages/linux/binwalk.md

```diff
@@ -0,0 +1,28 @@
+# Binwalk
+
+> Firmware Analysis Tool.
+> More information: <https://github.com/ReFirmLabs/binwalk>.
+
+- Scan a binary file:
+
+`binwalk {{binary}}`
+
+- Extract files from a binary specifying the output directory:
+
+`binwalk --extract --directory {{output_directory}} {{binary}}`
+
+- Recursively extract files from a binary limiting the recursion depth to 2:
+
+`binwalk --extract --matryoshka --depth 2 {{binary}}`
```

Comment:
Perhaps `2` could be in `{{token_syntax}}`?

```suggestion
`binwalk --extract --matryoshka --depth {{2}} {{binary}}`
```

Reply:
Yes, you're right. I didn't know if I had to consider it as a token while 2 was a fixed value in the example statement.

Reply:
In this case I think it should be, because it's something that the user can change and not affect the actual operation performed.